(func try try try catch_all end catch_all end catch_all end )